pkg/wrtcconn/adapter.go

```diff
@@ -794,7 +788,11 @@ func (a *Adapter) Open() (chan string, error) {
 									}
 								}()
 
-								c.candidates <- webrtc.ICECandidateInit{Candidate: string(candidate.Payload)}
+								c.candidates <- webrtc.ICECandidateInit{
```

Comment:
when SDPMid and SDPMLineIndex is null, js AddICECandidate will throw error, so I add the default value